src/lib/rest/HttpStatusCode.h

```diff
@@ -28,6 +28,8 @@
 * Author: developer
 */
 
+#define ERROR_STRING_PARSERROR  "ParseError"
```

Comment:
Not sure if this is the best place for this #define. I have put in HttpStatusCode.h because in some sense it is related with responses. 

Alternatives places are welcome!